src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import { Switch, Route } from 'react-router-dom'
+import Shop from '../../screens/Shop/Shop'
```

Comment:
yeah, the rules should be if something is only needed for container component it goes there but some more generic extract to components even if it's only used one time in time of creation 

i would recommend something around this lines ...
( also when you go with absolute paths then it's easy to move stuff around  )
```
 ./src
   ./lib
      ./data
      ./images
      ./services ( some common services for api handling )
      ./utils ( utils needed cross components/containers like DomUtil etc... )
   ./components
      ./Button
         Button.jsx
   ./containers 
      ./home
         Home.jsx  
```
Not to `dist`, this folder should be removed on every build, and re created from start and shuld be on root level of repo as `./src`